### DISCUSSION TITLE
Minecraft Java Edition (with ARM Support)

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#5] [General]
# Minecraft Java Edition (with ARM Support)

A community-driven list of apps and games that work, are emulated or don't work on Windows on ARM

https://windowsonarm.org/

**Comment by badbob001** [comment]
Use the Legacy Launcher from minecraft.net to launch Java Edition. It will use a built-in arm64 version of java.

**Reply by OpenSource03** [reply]
My suggestion is to simply use https://prismlauncher.org/download/, which should install the appropriate Java version by default.

I've updated the community url to point out to that.

**Reply by Lampadina17** [reply]
there is also the morpheus launcher as a valid alternative

**Comment by Omoore3** [comment]
I'm using prism launcher myself. Problem is, not all versions of the game work with the newer versions of java that actually have aarch64 ports, so say for example, Minecraft 1.8.9 doesn't work. What did work though was installing java 8 on WSL, and running the game through there. It's a little laggy, and the curse doesn't lock to the center of the screen, creating a pretty annoying bug, but it's playable from a very casual perspective (if you're okay with sub 20fps and aren't concerned with being fast or accurate with your movement at all).

I'm sure there's more that can be done to make the experience a little better, but I'm just reporting that it is possible to run all versions of the game through WSL.
